src/v2/Components/Artwork/Details.tsx

```diff
@@ -49,35 +40,33 @@ const ArtistLine: React.FC<DetailsProps> = ({
 
   if (cultural_maker) {
     return (
-      <TruncatedLine>
-        <Text variant={tokens.variant}>{cultural_maker}</Text>
-      </TruncatedLine>
+      <Text variant={tokens.variant} overflowEllipsis>
+        {cultural_maker}
+      </Text>
     )
   }
 
   if (artists && artists.length) {
     return (
-      <TruncatedLine>
-        <Text variant={tokens.variant}>
-          {artists
-            .reduce((acc, artist, index) => {
-              // @ts-expect-error STRICT_NULL_CHECK
-              return acc.concat([
-                ", ",
-                <ConditionalLink
-                  includeLinks={includeLinks}
-                  // @ts-expect-error STRICT_NULL_CHECK
-                  href={artist.href}
-                  key={index}
-                >
-                  {/* @ts-expect-error STRICT_NULL_CHECK */}
-                  <Text variant={tokens.variant}>{artist.name}</Text>
-                </ConditionalLink>,
-              ])
-            }, [])
-            .slice(1)}
-        </Text>
-      </TruncatedLine>
+      <Text variant={tokens.variant} overflowEllipsis>
+        {artists
+          .reduce((acc, artist, index) => {
```

Comment:
Off topic but I have a strong preference for extracting these types of values into a local constant. I pulled up this file and found it tricky to read this component but I now see there are three ways through:

* if there's a cultural maker, return that and bail on the rest of the function
* if there's no artists, bail
* if there are artists, join them with spaces and run them

I've got a local branch with this extracted and cleaned up a bit - you cool with me following up here after your works lands?

Reply:
Yah that's fine. I also intend to go back over this component with the updated RouterLink as there's more nodes to be removed. Feel free to PR me once this is merged.